frame/metadata/src/lib.rs

```diff
@@ -300,6 +300,11 @@ pub enum StorageEntryType {
 		value: DecodeDifferentStr,
 		key2_hasher: StorageHasher,
 	},
+	NMap {
```

Comment:
Just another note. You can actually include this, without use and without bumping. However, the new enum entry needs to be right at the end as to not affect existing encoding. 

This means we can bump the metadata version when we actually do use it in assets.